src/integrations/magni.ts

```diff
@@ -21,3 +28,36 @@ export const postFeedback = async (
       'Content-Type': 'application/json',
     },
   });
+
+  if (!res.ok) throw new ValidationError(await res.text());
```

Comment:
Refactored the `postFeedback` to check for an `ok` response.

Reply:
Good point. Let me check with the platform team what will be the supposed expectation gere